### PR TITLE
[docs] Fix kwarg docstring

### PR DESCRIPTION
**Pull request by stevhliu**
Fixes the format of the `kwargs` docstrings.

### CODE REVIEW

File: src/accelerate/utils/deepspeed.py [hunk 1]
```diff
@@ -234,7 +234,7 @@ class DummyOptim:
             parameter groups
         weight_decay (float):
             Weight decay.
-        **kwargs:
+        **kwargs (additional keyword arguments, *optional*):
```
Comment (muellerzr) [inline src/accelerate/utils/deepspeed.py:237]: Does the `**` make a difference?

**Review by muellerzr — APPROVED** [review]
Thanks for doing this! Just one question

File: src/accelerate/utils/deepspeed.py (hunk 1 above)
Comment (stevhliu) [reply src/accelerate/utils/deepspeed.py:237]: Nope, not in terms of formatting! `**kwargs` still shows up as expected. We can remove the `**` if you want :)

Screenshot 2024-03-26 at 12 39 09 PM

Comment (stevhliu) [reply src/accelerate/utils/deepspeed.py:237]: I got you 👍

**Pull request merged**